packages/storybook/src/nimble/select/select-opened-matrix.stories.ts

```diff
@@ -28,17 +29,53 @@ type FilterModeState = (typeof filterModeStates)[number];
 const placeholderStates = [false, true] as const;
 type PlaceholderState = (typeof placeholderStates)[number];
 
+const groupedStates = [false, true] as const;
+type GroupedState = (typeof groupedStates)[number];
+
+const optionsOutsideGroupStates = [false, true] as const;
+type OptionsOutsideGroupState = (typeof optionsOutsideGroupStates)[number];
+
 // prettier-ignore
 const component = (
     [position, positionStyle]: PositionState,
     filterMode: FilterModeState,
-    placeholder?: PlaceholderState
+    placeholder?: PlaceholderState,
```

Comment:
What do you think of moving these parameters into an options object? That would make it easier to not provide some of them and it would make the calls more readable because you'd see the names of the parameters at the call.

`component({postionState: positionStates[0], filterMode: FilterMode.standard, placeholder: true, grouped: true})`